src/components/App.tsx

```diff
@@ -2399,7 +2408,7 @@ class App extends React.Component<AppProps, AppState> {
       this.hitLinkElement!,
       this.state,
       [lastPointerDownCoords.x, lastPointerDownCoords.y],
-      this.isMobile,
+      this.isMobile || isIPad,
```

Comment:
Based on some testing, my answer is, yes, however at this time isIPad is not reliable.
I think instead of isMobile and isIPad we should have something like isTouchScreen or similar. I would detect it similar to penMode.

Reply:
Hmm not sure how this is related to fix though, coz `isMobile` check is just to make sure we allow clicking whole element in view mode in non mobile devices, whats the value of `window.navigator.platform` for ipad ?